app/jobs/audit_fixity_check_job.rb

```diff
@@ -3,23 +3,36 @@
 class AuditFixityCheckJob < ApplicationJob
   queue_as :default
 
-  def perform(package:, user:, audit:, **kwargs)
+  # TODO: Taking a bag as `storage` here is a testing concession; this should
+  # be cleaned up such that the package and its storage are not separately
+  # passed. We also need a better naming distinction between the artifact and
+  # the storage of it. That may fall out of the other refactoring.
+  def perform(
+    package:,
+    user:,
+    audit:,
+    storage: Services.storage.for(package),
+    mailer: AuditMailer
+  )
     @package = package
     @user = user
     @audit = audit
+    @mailer = mailer
+    @storage = storage
 
-    save_event(*validate_bag(**kwargs))
+    save_event(*validate)
   end
 
-  # TODO: This should just receive a bag.
-  def validate_bag(storage: Services.storage, bag: storage.for(package), mailer: AuditMailer)
+  private
+
+  def validate
     begin
-      if bag.valid?
+      if storage.valid?
```

Comment:
This is definitely confusing, but as you mention in the TODO above this is likely to be cleaned up later.

Reply:
Per discussion, we agree that "storage" is not quite right. I suspect that, as we pull the notion of an artifact up, that this may become the package itself; signifying that it's an in-storage representation (incoming or preserved) of a version of an artifact.